pkg/security/probe/constantfetch/fallback.go

```diff
@@ -209,6 +209,8 @@ func getSignalTTYOffset(kv *kernel.Version) uint64 {
 		ttyOffset = 400
 	case kv.IsInRangeCloseOpen(kernel.Kernel5_0, kernel.Kernel5_1):
 		ttyOffset = 408
+	case kv.IsInRangeCloseOpen(kernel.Kernel5_4, kernel.Kernel5_5) && kv.Code.Patch() >= 107:
```

Comment:
Do you have a source for this ? Can you maybe add a comment explaining why it starts at path 107

Reply:
This is related to:
```
On Ubuntu focal:
    - with linux-image-5.4.0-107-generic (kernel 5.4.0-107.121): fix fix "tty_offset" from 408 to 400.
```
I didn't check all available kernel for ubuntu focal, I just assumed that the previous offset (408) for 5.4 kernel was intended for kernel previous to patch 107, but the divergence patch could be minor than 107.

I will investigate more further this one to be more accurate

Reply:
After discussion, as it's totally pain to validate the change for all focal kernels, and as it should not matter (BTF being available), we choose to remove this change for now.